test/stress/s3benchmark.cc

```diff
@@ -0,0 +1,266 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+#include <climits>
+#include <cmath>
+#include <cstring>
+#include <string>
+
+
+#include "hash.h"
+#include "platform.h"
+#include "prng.h"
+#include "smalloc.h"
+#include "upload.h"
+#include "upload_s3.h"
+#include "upload_spooler_definition.h"
+#include "util/posix.h"
+#include "util/string.h"
+
+using namespace std; // NOLINT
+
+struct TestDataChunk {
```

Comment:
It would be good to initialize the struct in the default constructor (`data(NULL), size(0)`)

Reply:
Remember to add a default constructor